konfera/models/ticket.py

```diff
@@ -58,4 +58,12 @@ def save(self, *args, **kwargs):
             order = Order(price=self.type.price, discount=discount, status=AWAITING, purchase_date=timezone.now())
             order.save()
             self.order = order
-        super(Ticket, self).save(*args, **kwargs)
+            super(Ticket, self).save(*args, **kwargs)
+        else:
+            super(Ticket, self).save(*args, **kwargs)
```

Comment:
Please update the code so it calls super just once at the end of this function.

In else block we can just call `order.recalculate_tickets()`